src/selectors/publish.js

```diff
@@ -69,18 +70,25 @@ export const getRemoveProfileInformation: Selector<RemoveProfileInformation | nu
   getHiddenLocalTracksByPid,
   getGlobalTracks,
   getLocalTracksByPid,
+  getHasPreferenceMarkers,
   (
     checkedSharingOptions,
     profile,
     committedRange,
     hiddenGlobalTracks,
     hiddenLocalTracksByPid,
     globalTracks,
-    localTracksByPid
+    localTracksByPid,
+    hasPreferenceMarkers
   ) => {
     let isIncludingEverything = true;
-    for (const value of Object.values(checkedSharingOptions)) {
-      isIncludingEverything = isIncludingEverything && value;
+    for (const prop in checkedSharingOptions) {
+      // Do not include preference values checkbox if it's hidden.
```

Comment:
maybe write clearly that if it's hidden it's always false, even though it's not taken into account, otherwise we don't really understand the comment.

Reply:
It actually depends on the `profile.meta.updateChannel` because when update channel is nightly we include all the information and to do that, we switch all the checkboxes to `true` for the profiles that are not nightly, we are keeping them false to remove everything by defauly. Updated the comment to explain that.